Mathlib/Data/Real/GoldenRatio.lean

```diff
@@ -90,6 +90,10 @@ theorem gold_sub_goldConj : φ - ψ = Real.sqrt 5 := by
   ring
 #align gold_sub_gold_conj gold_sub_goldConj
 
+theorem gold_exp (n : ℕ) : φ ^ (n + 2) - φ ^ (n + 1) = φ ^ n := by
+  rw [goldenRatio]; ring_nf; norm_num; ring_nf
```

Comment:
Note, we use `ring` when the tactic closes the goal. `ring` and `ring_nf` don't do *quite* the same thing.
```suggestion
  rw [goldenRatio]; ring_nf; norm_num; ring
```